.travis.yml

```diff
@@ -17,8 +17,9 @@ install:
   - conda update conda
 
   # Install dependencies
-  - conda create -n test -c conda-forge python=$TRAVIS_PYTHON_VERSION pytest mock moto boto3
+  - conda create -n test -c conda-forge python=$TRAVIS_PYTHON_VERSION pip
   - source activate test
+  - pip install -r requirements.txt -r test_requirements.txt
```

Comment:
What was not being installed previously?

Reply:
Nothing, I just moved the installation from `conda` *outside* the environment to `pip` *inside* the environment. The reason for this is that we now can change the requirements in only one place and won't have to worry about updating the Travis file as well.